src/fluree/db/peer/http_api.clj

```diff
@@ -612,69 +613,74 @@
-                        acc)))
-        raft'     (-> raft
-                      (dissoc :cmd-queue :new-db-queue :networks)
-                      (assoc :cmd-queue cmd-q
-                             :new-db-queue new-db-q
-                             :networks nw-data))
-        state     (-> (txproto/-state (:group system))
-                      (select-keys [:snapshot-term
-                                    :latest-index
-                                    :snapshot-index
-                                    :other-servers
-                                    :index
-                                    :snapshot-pending
-                                    :term
-                                    :leader
-                                    :timeout-at
-                                    :this-server
-                                    :status
-                                    :id
-                                    :commit
-                                    :servers
-                                    :voted-for
-                                    :timeout-ms])
-                      (assoc :open-api open-api?)
-                      (assoc :raft raft')
-                      (assoc :svr-state svr-state))]
-    {:status  200
-     :headers {"Content-Type" "application/json; charset=utf-8"}
-     :body    (json/stringify-UTF8 state)}))
+  (let [deferred (d/deferred)]
+    (async/go
+      (try
+        (let [open-api? (open-api? system)
+              raft      (-> system :group :state-atom deref (dissoc :private-key))
+              {:keys [cmd-queue new-db-queue networks leases]} raft
+              instant   (System/currentTimeMillis)
+              cmd-q     (loop [[cq & r] cmd-queue
+                               acc []]
+                          (if cq
+                            (let [[k v] cq
+                                  acc* (into acc [{(keyword k) (count v)}])]
+                              (recur r acc*))
+                            acc))
+              new-db-q  (loop [[nq & r] new-db-queue
+                               acc []]
+                          (if nq
+                            (let [[k v] nq
+                                  acc* (into acc [{(keyword k) (count v)}])]
+                              (recur r acc*))
+                            acc))
+              nw-data   (remove-deep [:private-key] networks)
+              svr-state (when-let [servers (into [] (:servers leases))]
+                          (loop [[server & r] servers
+                                 acc []]
+                            (if-let [item (second server)]
+                              (recur r (into acc [{:id      (:id item)
+                                                   :active? (> (:expire item) instant)}]))
+                              acc)))
+              raft'     (-> raft
+                            (dissoc :cmd-queue :new-db-queue :networks)
```

Comment:
You can remove this line if you want because the `assoc` call below will overwrite the values for keys already present in the map

Reply:
made changes (and verified)